.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,130 @@
+# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
+
+name: "Continuous Integration"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - "master"
+
+jobs:
+  coding-standards:
+    name: "Coding Standards"
+
+    runs-on: "ubuntu-latest"
+
+    strategy:
+      matrix:
+        php-version:
+          - "7.2"
```

Comment:
Not sure, do you think it is necessary to actually run `phpcs` on `7.3`, `7.4`, and `8.0`?

Reply:
I'm not sure, but maybe when a sniff belongs to code only valid beginning with a certain PHP version.

Reply:
I believe the target PHP version should be expressed in the configuration of the standard. See an [example](https://github.com/doctrine/dbal/blob/ed4ac2afff18300a7f6fe5541319acfb2d2995f2/phpcs.xml.dist#L30-L35).

Static analysis should not produce different results with different runtimes, so more than one PHP version shouldn't be necessary.

Reply:
With previous discussion regarding version-dependent functionality (e.g. native type hints, see https://github.com/doctrine/coding-standard/pull/144), testing gets quite complicated. The main build should test against the lowest supported version, which is 7.2.

However, our standard produces different results when running on different PHP versions. There is a mechanism with a patch file that allows for running on PHP 7.4 as well, but I feel that maintaining this mechanism for multiple versions will be quite cumbersome. However, changing this would be a policy change that should not be discussed in this PR.

So for now I'm fine running this only on 7.2. Before replacing our travis-ci builds with GitHub Actions, we should discuss the issue of running on separate versions separately.